pkg/machine/qemu/machine.go

```diff
@@ -622,10 +622,15 @@ func CheckActiveVM() (bool, string, error) {
 // startHostNetworking runs a binary on the host system that allows users
 // to setup port forwarding to the podman virtual machine
 func (v *MachineVM) startHostNetworking() error {
-	// TODO we may wish to configure the directory in containers common
-	binary := filepath.Join("/usr/libexec/podman/", machine.ForwarderBinaryName)
-	if _, err := os.Stat(binary); err != nil {
-		return err
+	// MacOS does not have /usr/libexec so we look in the executable
+	// paths.
+	binary, err := exec.LookPath(machine.ForwarderBinaryName)
```

Comment:
I think it would be better to reverse this, since we expect ForwarderBinaryName to be in /usr/libexec/podman by default.
Check for that file and then check the path.  That way we don't accidentally get the wrong executables.

An other option would be to just create a function that cehcks for LookPath only in Darwin.

Reply:
The second options sounds more robust for me. Then shall we create “machine_darwin.go” and “machine_not_darwin.go” then separate the implementation?

Reply:
I'd vote for Stat the default then search the ExecPath.  I'd consider that desired behavior on any platform.  It may be appropriate to emit a WARN or DEBUG log message when reverting to the ExecPath.  (Level of message _could_ be platform dependent). And no, I don't know the Podman logging architecture -- yet.

Reply:
Something like:

```
        binary := filepath.Join("/usr/libexec/podman/", machine.ForwarderBinaryName)
        if _, err := os.Stat(binary); err != nil {
               if binary, err = exec.LookPath(machine.ForwarderBinaryName); err != nil {
                       return err
               }
        }
```